doc/mundo.txt

```diff
@@ -241,23 +253,32 @@ for a faster playback.
 Default: 60
 
 ------------------------------------------------------------------------------
-3.13 g:mundo_mirror_graph                                *mundo_mirror_graph*
+3.14 g:mundo_mirror_graph                                *mundo_mirror_graph*
 
 Set this to 0 to align the graph to the left; set to 1 to align to the right.
 
 Default: 1 (mirror graph)
 
 ------------------------------------------------------------------------------
-3.14 g:mundo_inline_undo                                 *mundo_inline_undo*
+3.15 g:mundo_inline_undo                                 *mundo_inline_undo*
 
 When enabled, a small one line diff is displayed to the right of the graph undo.
 Although not as detailed as a full diff provided in the preview window, it
 provides a quick summary of the diff w/o having to navigate.
 
 Default: 0 (no inline graph)
 
+-----------------------------------------------------------------------------
+3.16 g:mundo_enable_inline_delay                   *mundo_enable_inline_delay*
+
+When this option is set, inline diff rendering will be delayed until a preview
+diff is rendered. Use this to speed up undo tree traversal when inline diffs
+and delayed automatic previews are both enabled.
+
+Default: 0 (disabled)
+
 ------------------------------------------------------------------------------
-3.15 g:mundo_return_on_revert                         *mundo_return_on_revert*
+3.17 g:mundo_return_on_revert                         *mundo_return_on_revert*
 
 Set this to 0 to keep focus in the Mundo window after a revert.
```

Comment:
please add a summary of this feature the the changelog at the bottom of this file.

Reply:
Done. I'm assuming the project roughly follows [semantic versioning](https://semver.org/)? If so, it might be worth making a note of this in the documentation and/or README.

Reply:
Yeah, it *roughly* follows semantic versioning - or at least I'd like us to try to do that. If you have it in you to add a note about that, please do!